crates/starknet-server/tests/get_transaction_receipt_by_hash.rs

```diff
@@ -148,7 +148,65 @@ mod get_transaction_receipt_by_hash_integration_tests {
     }
 
     #[tokio::test]
-    async fn get_declare_v1_transaction_receipt_by_hash_happy_path() {
+    async fn reverted_invoke_transaction_receipt() {
+        let devnet = BackgroundDevnet::spawn().await.expect("Could not start Devnet");
+
+        let (private_key, account_address) = devnet.get_first_predeployed_account().await;
+
+        // constructs starknet-rs account
+        let signer = LocalWallet::from(SigningKey::from_secret_scalar(private_key));
+
+        let mut predeployed_account = SingleOwnerAccount::new(
+            devnet.clone_provider(),
+            signer,
+            account_address,
+            chain_id::TESTNET,
+            ExecutionEncoding::Legacy,
+        );
+
+        // `SingleOwnerAccount` defaults to checking nonce and estimating fees against the latest
+        // block. Optionally change the target block to pending with the following line:
+        predeployed_account.set_block_id(BlockId::Tag(BlockTag::Pending));
+
+        let transfer_execution = predeployed_account.execute(vec![Call {
+            to: FieldElement::from_hex_be(ERC20_CONTRACT_ADDRESS).unwrap(),
+            selector: get_selector_from_name("transfer").unwrap(),
+            calldata: vec![
+                FieldElement::ONE,
+                FieldElement::from_dec_str("1000000000").unwrap(),
+                FieldElement::ZERO,
```

Comment:
What do these magical numbers represent? Also, since this is the second time I'm seeing this, could this somehow be extracted?

Reply:
[34e5e89](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/213/commits/34e5e89f969f962646ff3d3afd67789c7efef3b1)